prisma/schema.prisma

```diff
@@ -58,3 +86,15 @@ model Feedback {
     updatedAt    DateTime @updatedAt
     user         User?    @relation(fields: [userUUID], references: [uuid], onDelete: Cascade)
 }
+
+model ProfilePicture {
+    id        Int      @id @default(autoincrement())
+    uuid      String   @unique @default(uuid())
+    userUUID  String   @unique
+    thumbnail Bytes    @db.ByteA
+    picture   Bytes    @db.ByteA
+    createdAt DateTime @default(now())
+    updatedAt DateTime @updatedAt
+    // tähän Person:iin ei ole laitettu relaatiota, koska relaatio tulee jo Person-modelista
+    Person    Person?
+}
```

Comment:
`ByteA`:n käyttö vaati tuon `Bytes`in ja sitten `@db.ByteA`. Ainakin näin netissä neuvottiin, onko tuo ok?

Myöskään `Person`:lle ei ole tehty relaatiota tässä Model:ssa

Reply:
> `ByteA`:n käyttö vaati tuon `Bytes`in ja sitten `@db.ByteA`. Ainakin näin netissä neuvottiin, onko tuo ok?

On ok, just näin!

> Myöskään `Person`:lle ei ole tehty relaatiota tässä Model:ssa

Tähän tulee muutos, ks. aikaisempi kommentti missä perustelin miksi linkitys Person-modeliin on parempi tehdä täällä.